### DISCUSSION TITLE
Logo in mail notifications

### DISCUSSION THREAD

**Discussion by PierreDurrr** [sct/overseerr#2128] [Help] [answered]
Don't know if it's a known issue or not, but there is no logo displayed in any mail notification sent by overseerr.
Application URL is set.

**Comment by TheCatLady** [comment, marked as answer]
This is most likely an issue with how you are proxying your Overseerr instance.  I would start by checking the Content Security Policy (CSP) and make sure it is configured appropriately to allow external embedding of the image.

If the notifications are being sent to a Gmail address, another possible issue is that Google's image proxy is being blocked.  If you use Cloudflare, overzealous firewall rules could also be the culprit.

**Comment by Robpollard** [comment]
I have the same problem. I had help setting up Overseerr and am thus a bit stumped on how to proceed. I use Gmail plus Cloudflare - any pointers on where to start the troubleshooting? If I open the blocked picture in a new tab, I get a 404 "The requested URL /proxy/2iQaKoPmebt0r7VLHtxvttcdcemGYRIEbvKAKp8FfrTRPz8LDHdFvg=s0-d-e1-ft was not found on this server"
